frontend/src/js/controllers/challengeHostTeamsCtrl.js

```diff
@@ -309,7 +309,7 @@
 
         vm.inviteOthers = function(ev, hostTeamId) {
             ev.stopPropagation();
-            // Appending dialog to document.body 
+            // Appending dialog to document.body
```

Comment:
@HargovindArora Please remove this change from this PR.

Reply:
@RishabhJain2018 Okay!